### DISCUSSION TITLE
Formily 1.x的<FormSpy> 的reducer函数会触发多次

### DISCUSSION THREAD

**Discussion by ggd543** [alibaba/formily#2425] [Q&A 1.x] [answered]
https://codesandbox.io/s/musing-dream-7kpgm?file=/index.js

用FormSpy 监听 LifeCycleTypes.ON_FORM_VALUES_CHANGE ,  每次表单内容发生变化时，reducer会触发2次

**Comment by janryWang** [comment, marked as answer]
因为1.x内核对数据的监听不是reactive的，做不到精确的数据响应，触发多次是避免不了的，你可以加个debounce解决
